notebooks/18.In_Depth-Trees_and_Forests.ipynb

```diff
@@ -136,6 +136,7 @@
     "from sklearn.model_selection import train_test_split\n",
     "from sklearn.tree import DecisionTreeClassifier\n",
     "from figures import plot_2d_separator\n",
+    "from figures import cm2\n",
```

Comment:
I'm not 100% sure I'm actually a fan of using the colormaps from the book (this is what this is) because they are non-standard. On the other hand, there is no good way to do this otherwise, I think :-/

Reply:
I change them to be tab10 standard

Reply:
the naming is not standard though...